views/tutors.html

```diff
@@ -30,6 +32,36 @@ <h3 class="d-inline" id="year-current" data-year='{{ @year }}'>{{ @year }} - {{@
         <check if="{{ @currentYear != @year }}">
             <button id="current-year" class="btn btn-info mt-3">Set Current Year</button>
         </check>
+        <br>
+
+        <!-- Filter buttons -->
+        <div class="mt-3">
+            <input type="hidden" value="{{ @status }}" id="status">
+            <check if="{{ @status == 'incomplete' }}">
+                <true>
+                    <button class="btn btn-primary filter">Incomplete</button>
+                </true>
+                <false>
+                    <a href="/tutors/{{ @year }}&incomplete"><button class="btn btn-secondary filter">Incomplete</button></a>
```

Comment:
just letting you know that you can make anchor tags look like buttons with bootstrap. basically <a> and <button> are treated the same in bootstrap.